services/src/main/java/io/scalecube/services/Microservices.java

```diff
@@ -620,13 +674,6 @@ private JmxMBeanBootstrap(Microservices microservices) {
       return Collections.singletonList(microservices.discovery().address().toString());
     }
 
-    @Override
-    public Collection<String> getGatewayAddresses() {
```

Comment:
Please add gateway binding info in JMX bean.

Reply:
Done